tests/api/v1/aspects/postWriters.js

```diff
@@ -65,7 +66,10 @@ describe('api: aspects: post writers', () => {
     })
     .then((tUsr) => {
       thirdUser = tUsr;
-      userNameArray.push(thirdUser.name);
```

Comment:
@shriramshankar Do we still need to keep the thirdUser variable? Its value doesn't seem to be used.